doc/specs/vulkan/chapters/initialization.txt

```diff
@@ -31,21 +31,28 @@ Vulkan commands as well; if this is done, then applications that use only
 the core Vulkan commands have no need to use fname:vkGetInstanceProcAddr.
 
 Function pointers to commands that don't operate on a specific instance can:
-be obtained by using this command with pname:instance equal to `NULL`. The
-following commands can: be accessed this way:
+be obtained by using this command with pname:instance equal to `NULL`. Only
+pointers to the following commands can: be obtained this way:
 
   * flink:vkEnumerateInstanceExtensionProperties
   * flink:vkEnumerateInstanceLayerProperties
   * flink:vkCreateInstance
 
-If pname:instance is a valid sname:VkInstance, function pointers to any
-commands that operate on pname:instance or a child of pname:instance can: be
-obtained. The returned function pointer must: only be called with a
-dispatchable object (the first parameter) that is a child of pname:instance.
+If pname:instance is a valid sname:VkInstance, then function pointer to any
+command from any of the following groups can: be obtained:
 
-If pname:pName is not the name of a core Vulkan command, or is an
-extension command for any extension not supported by any available layer or
-implementation, then fname:vkGetInstanceProcAddr will return `NULL`.
+  * core Vulkan commands
+  * commands of instance extensions enabled for pname:instance
+  * commands of device extensions enabled for a valid sname:VkDevice child
+    object of pname:instance
+
+Any other valid usage of fname:vkGetInstanceProcAddr command must:
+return `NULL`.
+
+If pname:instance is a valid sname:VkInstance and the command does return a
+valid (non-`NULL`) function pointer, then the returned function pointer
+must: only be called with a dispatchable object (the first parameter) being
+pname:instance or its child.
```

Comment:
I'm liking the table form. Defines what happens for all situations and feels less wordy. I'd just replace all the descriptive text with the table.